.ci/setup.sh

```diff
@@ -67,6 +67,9 @@ enable_nested_virtualization() {
 	aarch64)
 		info "CI running in bare machine"
 		;;
+	ppc64le)
```

Comment:
just a hint, to avoid repeating the same `info` message you could do:
```bash
aarch64 | ppc64le)
		info "CI running in bare machine"
		;;
```

Reply:
@marcov thanks for the hint .. updated the patch accordingly